src/ekat/io/ekat_yaml.cpp

```diff
@@ -1,43 +1,95 @@
+#include "ekat/util/ekat_meta_utils.hpp"
 #include "ekat/io/ekat_yaml.hpp"
 #include "ekat/ekat_assert.hpp"
 
 #include <yaml-cpp/yaml.h>
 
 #include <sstream>
 #include <fstream>
+#include <iomanip>
 
 namespace ekat {
 
+// These are the values we can parse.
+using values_t     = TypeList<bool,int,double,std::string>;
```

Comment:
Why the `snake_case` type names here? Just curious.

Reply:
I usually prefer `snake_case` for _local_ type names (i.e., aliases and typedefs). Not sure why I prefer that.